### ISSUE TITLE
Structure builder should automatically save

### ISSUE THREAD

**Issue by pjknowles** [molpro/iMolpro#30]
At the moment, it's easy to forget to save an edited structure.

**Comment by klauspdoll** [comment]
Yes, now I have seen the button "Save structure" and it works. Maybe mention this somewhere in the manual, or have a bigger button, whatever. ...

**Comment by pjknowles** [comment]
Following https://github.com/molpro/iMolpro/pull/366 , there is no structure builder any more.
